groups/nts/ntsa/ntsa_interest.t.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020-2023 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <ntsa_interest.h>
+#include <ntscfg_test.h>
+
+using namespace BloombergLP;
+
+//=============================================================================
+//                                 TEST PLAN
+//-----------------------------------------------------------------------------
+//                                 Overview
+//                                 --------
+//
+//-----------------------------------------------------------------------------
+
+// [ 1]
+//-----------------------------------------------------------------------------
+// [ 1]
+//-----------------------------------------------------------------------------
+
+NTSCFG_TEST_CASE(1)
```

Comment:
Is this test finished? maybe we can add some validation?

Reply:
I have committed an expanded test driver.